test_e2e.sh

```diff
@@ -49,8 +49,15 @@ function assert_file_eq() {
 
 # Build it
 echo "Building..."
-make >/dev/null
-GIT_SYNC=./bin/git-sync-amd64
+make container REGISTRY=e2e TAG=$(make version) >/dev/null
```

Comment:
For me the $(make version) is not silenced and therefore is assigned "Entering".
Would use $(make -s version) instead.